openapi-bal-service/src/main/java/io/ballerina/openapi/converter/service/OpenAPIResponseMapper.java

```diff
@@ -465,6 +465,16 @@ private void setCacheConfigField(CacheConfigAnnotation cacheConfig, String field
         if (array.memberTypeDesc().kind() == SIMPLE_NAME_REFERENCE) {
             handleReferenceResponse(operationAdaptor, (SimpleNameReferenceNode) array.memberTypeDesc(),
                     schemas02, apiResponses, customMediaPrefix, headers);
+        } else if (array.memberTypeDesc().kind() == QUALIFIED_NAME_REFERENCE) {
+            Optional<ApiResponses> optionalAPIResponses =
+                    handleQualifiedNameType(new ApiResponses(), customMediaPrefix, headers, apiResponse,
+                            (QualifiedNameReferenceNode) array.memberTypeDesc());
+            if (optionalAPIResponses.isPresent()) {
+                ApiResponses responses = optionalAPIResponses.get();
+                updateResponseWithArraySchema(responses);
+                apiResponses.putAll(responses);
+            }
+            return Optional.of(apiResponses);
```

Comment:
I think we don't need this return statement since we have a return after the `else` condition.

Reply:
yeah , will remove